docs/idl.md

```diff
@@ -327,3 +327,43 @@ will be `RecordWithConst::CONST_VALUE` in C++, `RecordWithConst.CONST_VALUE` in
 If comments are placed on top or inside a type definition, they will be converted to
 Javadoc / Doxygen compatible comments in the generated Java, C++, Objective-C and C++/CLI interfaces, or a
 Python docstring.
+
+## Deprecation Comments
+
+It's possible to add deprecation comments to IDL types and methods.
+
+The comment in the IDL file has to start the `# @deprecated reason/hing message` .
+
+This will generate the corresponding deprecation annotation in the target language.
+
+```idl
+# @deprecated Use someother interface
```

Comment:
If `someother` is meant to be a type we should follow Djinni's naming convention and write it as `some_other`.

Reply:
no, it's just something different, a placeholder for some error message, but maybe the fix is a good idea

Reply:
```suggestion
# @deprecated Use some_other interface
```